packages/block-library/src/embed/variations.js

```diff
@@ -339,4 +339,15 @@ const variations = [
 	},
 ];
 
+/**
+ * Add `isActive` function to all `embed` variations, if not defined.
+ * `isActive` function is used to find a variation match from a created
+ *  Block by providing its attributes.
+ */
+variations.forEach( ( variation ) => {
+	if ( variation.isActive ) return;
+	variation.isActive = ( { providerNameSlug } ) =>
```

Comment:
It would be more convenient when `variationAttributes` would be passed as the second param so people don't have to repeat hard coded values when they define this function in the variation object.

```js
isActive( blockAttributes, variationAttributes ) {
     return blockAttributes.providerNameSlug === variationAttributes.providerNameSlug;
},
```